tools/gfx/d3d12/render-d3d12.cpp

```diff
@@ -6296,14 +6298,14 @@ Result D3D12Device::createFence(const IFence::Desc& desc, IFence** outFence)
 Result D3D12Device::waitForFences(
     uint32_t fenceCount, IFence** fences, uint64_t* fenceValues, bool waitForAll, uint64_t timeout)
```

Comment:
Isn't the timeout here in nanoseconds (from the updated comment)? 

WaitForMultipleObjects is in milliseconds

https://docs.microsoft.com/en-us/windows/win32/api/synchapi/nf-synchapi-waitformultipleobjects